react/Header/Header.js

```diff
@@ -122,14 +121,7 @@ export default function Header({
             divider={divider}
           />
         </Hidden>
-        <Hidden print mobile className={styles.topBanner}>
```

Comment:
These `topBanner` styles should be split up, `Header` should still be responsible for the flex ordering as that is specific to this usage of PartnerSites. possible worth keeping the `Hidden` concept external as well? Thoughts @markdalgleish?

Reply:
Yeah, agreed—I think the flex ordering needs to be hoisted to a wrapping div in `Header`.

I think the `Hidden` concept belongs inside `PartnerSites` though, since the component itself isn't responsive, and is designed to never be printed globally either.

Reply:
Yep, just the ordering, since that's the only styling opinion that's specific to its usage within `Header`.

Reply:
👍